src-electron/db/query-impexp.js

```diff
@@ -674,8 +677,24 @@ WHERE
   } else if (reportingPolicy == dbEnums.reportingPolicy.prohibited) {
     attribute.reportable = false
   }
-
-  if (storagePolicy == dbEnums.storagePolicy.attributeAccessInterface) {
+  if (attributeId) {
+    forcedExternal = await queryUpgrade.getForcedExternalStorage(
+      db,
+      attributeId
+    )
+    storageOption = await queryUpgrade.computeStorage(
+      db,
+      cluster.name,
+      null,
+      storagePolicy,
+      forcedExternal,
+      attributeId
+    )
+  }
+  if (
+    storagePolicy == dbEnums.storagePolicy.attributeAccessInterface ||
+    storageOption == dbEnums.storageOption.external
```

Comment:
How can we have `storagePolicy == dbEnums.storagePolicy.attributeAccessInterface` but not `storageOption == dbEnums.storageOption.external`?

In other words, why are we not just checking `storageOption` here?

Reply:
I think this might've been silly debug code, thank you, will fix

Reply:
Unfortunately do to the scope of the variables this is needed. Will look into fixing it in the long term.
There are basically too many moving parts trying to use the same function. The solution might be just to break up "computeStorage" into multiple derivative functions. @brdandu this also would eliminate the need to pass null params. It seemed like a neat trick at the time but it is increasingly looking like the logic should be separated into different scenarios.